static/app/views/insights/browser/resources/utils/useResourceFilters.ts

```diff
@@ -22,17 +26,27 @@ export type ModuleFilters = {
   [BrowserStarfishFields.SPAN_OP]?: ResourceSpanOps;
   [BrowserStarfishFields.TRANSACTION]?: string;
   [BrowserStarfishFields.SPAN_DOMAIN]?: string;
+  [SpanMetricsField.USER_GEO_SUBREGION]?: SubregionCode[];
```

Comment:
Nit: Since `BrowserStarfishFields.USER_GEO_SUBREGION` was added, it feels like it would be more consistent to use that here. If the whole thing should be replaced with `SpanMetricsField` instead, that can be done in one shot later. Just a bit easier to follow IMO.

Reply:
Yeah I agree!